src/main/kotlin/org/fancy/memers/model/Block.kt

```diff
@@ -0,0 +1,28 @@
+package org.fancy.memers.model
+
+import org.hexworks.zircon.api.color.TileColor
+
+sealed class Block: Drawable
+
+class Empty(override val position: Position) : Block() {
+    override val symbol: Char
+        get() = ' '
```

Comment:
```suggestion
    override val symbol: Char = ' '
```
Кажется можно без `get()`?

Reply:
Разницы мало, только не хотелось стейта в блоках